uwuno/src/main/java/com/learning/uwuno/controllers/playerController.java

```diff
@@ -52,40 +36,82 @@ public String addPlayer(@RequestBody String json, @PathVariable String uid)
         }
     }
 
+    // GETS
+    // Return list of card available to player
+    @GetMapping(value = "rooms/{uid}/players/{pid}/cards")
+    public ArrayList<card> getPlayerCards(@PathVariable String uid, @PathVariable String pid) {
+        return containerService.getPlayer(uid, pid).getCardList();
+    }
+
     // PUTS
-    // Update player name in given room uid
-    @PutMapping(value = "rooms/{uid}/players")
```

Comment:
@achang1 If we are changing a state, the endpoint should always be at the specific url for the object.